ext/AvizExt/viz/clustering.jl

```diff
@@ -14,56 +16,91 @@ Visualize clustered time series of scenarios.
 Figure
 """
 function ADRIA.viz.clustered_scenarios(
-    data::AbstractMatrix,
+    data::NamedDimsArray,
     clusters::Vector{Int64};
     fig_opts::Dict=Dict(),
-    axis_opts::Dict=Dict()
-)
+    axis_opts::Dict=Dict(),
+    opts::Dict=Dict()
+)::Figure
     f = Figure(; fig_opts...)
     g = f[1, 1] = GridLayout()
 
-    ADRIA.viz.clustered_scenarios!(g, data, clusters; axis_opts=axis_opts)
+    ADRIA.viz.clustered_scenarios!(g, data, clusters; axis_opts=axis_opts, opts=opts)
 
     return f
 end
 function ADRIA.viz.clustered_scenarios!(
     g::Union{GridLayout,GridPosition},
-    data::AbstractMatrix,
+    data::NamedDimsArray,
     clusters::Vector{Int64};
-    axis_opts::Dict=Dict()
-)
-    # Ensure last year is always shown in x-axis
+    axis_opts::Dict=Dict(),
+    opts::Dict=Dict()
+)::Union{GridLayout,GridPosition}

```

Comment:
I just noticed this, please standardize order of kwargs to be consistent.

It should be in the order of: `opts`, `fig_opts`, and `axis_opts`